tools/src/test/java/org/cyclonedx/schema/XmlSchemaVerificationTest.java

```diff
@@ -13,39 +13,48 @@
  */
 package org.cyclonedx.schema;
 
-import org.cyclonedx.CycloneDxSchema;
-import org.cyclonedx.parsers.XmlParser;
-import org.junit.jupiter.api.DynamicTest;
-import org.junit.jupiter.api.TestFactory;
 import java.io.File;
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 
+import org.cyclonedx.parsers.XmlParser;
+import org.cyclonedx.Version;
+import org.junit.jupiter.api.DynamicTest;
+import org.junit.jupiter.api.TestFactory;
+
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 
 public class XmlSchemaVerificationTest extends BaseSchemaVerificationTest {
 
     @TestFactory
+    /**
+     * Generates a collection of dynamic tests based on the available XML files.
+     *
+     * @return Collection<DynamicTest> a collection of dynamic tests
+     * @throws Exception if an error occurs during the generation of the dynamic tests
+     */
     Collection<DynamicTest> dynamicTestsWithCollection() throws Exception {
         final List<String> files = getAllResources();
         final List<DynamicTest> dynamicTests = new ArrayList<>();
         for (final String file: files) {
             if (file.endsWith(".xml")) {
-                final CycloneDxSchema.Version schemaVersion;
+                final Version schemaVersion;
                 if (file.endsWith("-1.0.xml")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_10;
+                    schemaVersion = Version.VERSION_10;
                 } else if (file.endsWith("-1.1.xml")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_11;
+                    schemaVersion = Version.VERSION_11;
                 } else if (file.endsWith("-1.2.xml")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_12;
+                    schemaVersion = Version.VERSION_12;
                 } else if (file.endsWith("-1.3.xml")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_13;
+                    schemaVersion = Version.VERSION_13;
                 } else if (file.endsWith("-1.4.xml")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_14;
+                    schemaVersion = Version.VERSION_14;
                 } else if (file.endsWith("-1.5.xml")) {
-                    schemaVersion = CycloneDxSchema.Version.VERSION_15;
+                    schemaVersion = Version.VERSION_15;
+                } else if (file.endsWith("-1.6.xml")) {
```

Comment:
this file seams to be slipped in by accident. it is irrelevant for the java upgrade.
please remove it and create an extra PR for it

Reply:
In this case, I think you are wrong: upgrading from [`cyclonedx-core-java-8.0.3`](https://github.com/CycloneDX/cyclonedx-core-java/releases/tag/cyclonedx-core-java-8.0.3) to [`cyclonedx-core-java-9.0.2`](https://github.com/CycloneDX/cyclonedx-core-java/releases/tag/cyclonedx-core-java-9.0.2) has an impact on the class `CycloneDxSchema`: some of its content has been moved to a new file.

The `public enum Version` has moved from file `src/main/java/org/cyclonedx/CycloneDxSchema.java` in v8.0.3 to file `src/main/java/org/cyclonedx/Version.java` in v9.0.2.

For **cyclonedx-core-java-9.0.2**: there is a new file `Version.java`: 
https://github.com/CycloneDX/cyclonedx-core-java/blob/cyclonedx-core-java-9.0.2/src/main/java/org/cyclonedx/Version.java

If you only modify the `pom.xml` to upgrade from 8.0.3 to 9.0.2, then the Java tests will fail, as they need the content from the new file `src/main/java/org/cyclonedx/Version.java` that you get from `import org.cyclonedx.Version;`.

If you want me to do an extra PR for this, please elaborate because I don't understand how to do this.